hw2/04-background.js

```diff
@@ -1 +1,42 @@
-// Add your code here
+const body = document.getElementById("viewport");
+const timerValue = document.getElementById("timerInput");
+const timerButton = document.getElementById("button");
+let interval = null;
+let timerOn = false;
+
+function generateColorPalette() {
+  const r = Math.floor(Math.random() * 200);
+  const g = Math.floor(Math.random() * 200);
+  const b = Math.floor(Math.random() * 200);
+  const colorCode = `rgb(${r}, ${g}, ${b})`;
+  return colorCode;
+}
+
+function nextBackgroundColor() {
+  const colorCode = generateColorPalette();
+  body.style.backgroundColor = colorCode;
+}
+
+function initiateColorChange() {
```

Comment:
As I already mentioned in Kobe's PR that adding a error handling will notify the user if they give a wrong input.

Reply:
Since the input type is number and I have given a minimum value of 1, there is no option for a user to input any character except a number, that is why I did not include an additional validation for this.